recipes/ndarray_listener/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ndarray_listener" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "95d637109bbb647ad54fb57b982f9881d4501394395e82b0e1f484657ac29ec1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+  run:
+    - python
+    - numpy
+    - pytest
```

Comment:
Not trying to bikeshed this. Though would like to give some context.

If one `pip`-installed `numpy`, for example, it would not install `nose`. Even though `nose` is required to run `numpy.test()`, they still ship without `nose`. Continuum does the same with their `conda` package of `numpy` and so do we.

This is a pretty standard Python practice to have testing dependencies be an additional set of optional dependencies that user can install if they wish. That being said, handle this as you see fit.

Side note: There is no need to include your run time dependencies in your [test dependencies]( https://github.com/Horta/ndarray_listener/blob/e6887e3071d4c0ab4b5d12fb14b2bbfc8f5e2fd8/setup.py#L18 ) as they will be included automatically.